### DISCUSSION TITLE
Desktop: Avoid highway and toll not taken into account

### DISCUSSION THREAD

**Discussion by PymPC** [devemux86/cruiser#608] [Q&A] [answered]
Hello,
Sorry for this English that I'm not talking about, thank you online translators!.
While I use Cruiser to prepare my travel itteryes in countries whose cards are more precise (for example Iceland and its tracks), I wanted to prepare my itinerary to return from a French city (Piriac sur mer to Paris) . In the parameters of the route, I made sure to remove the highways and toll routes. That it was not my surprise to see that Cruiser made me take a highway on almost the entire route.
Is it a bug?
I use Cruiser (V4.0.2) on an office PC under Linux Manjaro Gnome.
I will try it during my trip to Iceland where many tracks await me ....

**Comment by devemux86** [comment, marked as answer]
@PymPC thanks for the report.

> I use Cruiser (V4.0.2) on an office PC under Linux Manjaro Gnome.

You use desktop Cruiser. _What routing service have you selected?_

You can select OpenRouteService or Valhalla that have these options.

---

Or use [Android Cruiser](https://play.google.com/store/apps/details?id=gr.talent.cruiser) that offers many and detailed routing options.

On desktop it is also better to use the router websites:
- [BRouter](https://brouter.de/brouter-web/)
- [OpenRouteService](https://maps.openrouteservice.org/)
- [Valhalla](https://valhalla.openstreetmap.de/)

Then you can import the result on Android Cruiser for navigation. :slightly_smiling_face:

**Comment by PymPC** [comment]
Thank you for this quick answer
Indeed I used Graphopper. I tested grazing but with this service the passing points on certain tracks cannot be used (alert message indicating that the point is too far from a road ...).
OpenRouteService or Valhalla seems better worked.
In any case thank you, I will test the Android version. I cannot get used to preparing a route on such a small screen .....
